### PR TITLE
[Release] version 2.0.1

### PR DESCRIPTION
**Pull request by kittinunf**
## Description

This PR will release version 2.0.1

## Type of change

Check all that apply

- [x] Release v2.0.1 🎉 

## How Has This Been Tested?

N/A

## Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new compiler warnings

### CODE REVIEW

**Comment by SleeplessByte** [comment]
Lgtm

File: fuel-coroutines/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ The coroutines extension package for [`Fuel`](../README.md).
 
 You can [download](https://bintray.com/kittinunf/maven/Fuel-Android/_latestVersion) and install `fuel-coroutines` with `Maven` and `Gradle`. The android package has the following dependencies:
 * `fuel:fuel:<same-version>`
-* Kotlin: 1.3.0
+* Kotlin: 1.3.20
 * KotlinX Coroutines: 1.0.1
```
Comment (iNoles) [inline fuel-coroutines/README.md]: It should be 1.1.1

File: fuel/README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+[![Kotlin](https://img.shields.io/badge/Kotlin-1.3.20-blue.svg)](https://kotlinlang.org)
 
 The core package for [`Fuel`](../README.md). The documentation outlined here touches most subjects and functions but is not exhaustive. 
 
 ## Installation
 
 You can [download](https://bintray.com/kittinunf/maven/Fuel-Android/_latestVersion) and install `Fuel` with `Maven` and `Gradle`. The core package has the following dependencies:
-* Kotlin - 1.3.10
+* Kotlin - 1.3.20
 * KotlinX Coroutines - 1.0.1
```
Comment (iNoles) [inline fuel/README.md]: KotlinX Coroutines: 1.1.1

File: fuel-coroutines/README.md (hunk 1 above)
Comment (NikkyAI) [inline fuel-coroutines/README.md]: should be version 1.1.1
why can we not have variables in markdown files again ?

Comment (kittinunf) [reply fuel-coroutines/README.md]: How can we do that?

**Pull request merged**